spec/swagger.yaml

```diff
@@ -2009,18 +2009,37 @@ paths:
 
   /context-api/v2/projects/{projectId}/contexts/upload-and-match-async:
     post:
-      summary: Upload HTML and match strings to create context bindings
+      summary: Upload HTML file and match strings to create context bindings
       description: |-
-        Uploads a new HTML context and asynhronously matches it with strings in the project.
-
-        Equivalent to calling:
-        - `POST /projects/{projectId}/contexts`
-        - `POST /projects/{projectId}/contexts/{contextUid}/match/async`
+        Uploads a new HTML context and asynhronously matches it with strings.
+        Strings to match are retrieved from the project specified by the `projectId` and 
+        can further be filtered by optional `matchParams` parameter.
```

Comment:
(A bit of word re-order and adding "the" to the sentence.): .....and can be filtered further by the optional `matchParams` parameter.